src/main/java/graphql/schema/validation/ObjectsImplementInterfaces.java

```diff
@@ -1,14 +1,6 @@
 package graphql.schema.validation;
 
-import graphql.schema.GraphQLArgument;
-import graphql.schema.GraphQLFieldDefinition;
-import graphql.schema.GraphQLInterfaceType;
-import graphql.schema.GraphQLList;
-import graphql.schema.GraphQLNonNull;
-import graphql.schema.GraphQLObjectType;
-import graphql.schema.GraphQLOutputType;
-import graphql.schema.GraphQLType;
-import graphql.schema.GraphQLUnionType;
+import graphql.schema.*;
```

Comment:
Can you please fix up * imports to be how they were before

Reply:
Done! Apologies for the churn, my editor automatically updated the imports.